src/components/notification/notification.vue

```diff
@@ -34,17 +71,31 @@ module.exports = {
     display: inline-flex;
     align-items: center;
     color: var(--a-text-primary);
-    background: var(--a-base-9);
+    background: var(--a-background);
     border: 1px solid var(--a-base-8);
     border-radius: var(--a-border-radius);
     padding: 15px 20px;
     box-shadow: var(--a-box-shadow);
   }
 
   .apos-notification__indicator {
+    position: relative;
+    top: 1px;
     display: flex;
```

Comment:
Flex doesn't seem to do anything anymore. If using relative/1px method here, we should do the same on the tags component.

Reply:
removed